sdk/identity/identity/test/manual-integration/AzureArc/INSTRUCTIONS.md

```diff
@@ -0,0 +1,73 @@
+# Testing Identity in Azure Arc
+
+## Prerequisites
+
+- A non-Azure Windows or Linux VM.
+  - Must have NodeJS, NPM, and git installed.
```

Comment:
"VM" above seems a bit rough for the first sentence in this document. Let's spell Virtual Machine in this case!
"Must have" reads a bit rough to me. What about "The Virtual Machine must have Node.js, NPM and Git installed." - can we provide links to the download page of each one directly in there? Like `[Node.js](download-page-link)`